ftw/envoy-config.yaml

```diff
@@ -37,7 +37,7 @@ static_resources:
                             {
                               "rules": [
                                 {"inline": "Include ftw-config.conf"},
-                                {"inline": "Include coraza.conf-recommended"},
+                                {"inline": "Include coraza-ftw.conf"},
```

Comment:
We should still include coraza recommended for users right?

Can `ftw-config.conf` be modified to set the SecResponseBodyMimeType etc? It can control priority by doing the `Include coraza.conf-recommended` there instead of here, not sure if it would be before or after the include but one of them would work I think

Reply:
Makes sense! Done!